base/uk.ac.stfc.isis.ibex.synoptic/META-INF/MANIFEST.MF

```diff
@@ -14,7 +14,9 @@ Require-Bundle: org.eclipse.core.runtime,
  uk.ac.stfc.isis.ibex.instrument,
  uk.ac.stfc.isis.ibex.json;bundle-version="1.0.0",
  uk.ac.stfc.isis.ibex.configserver;bundle-version="1.0.0",
- uk.ac.stfc.isis.ibex.epics.switching
+ uk.ac.stfc.isis.ibex.epics.switching,
+ uk.ac.stfc.isis.ibex.devicescreens,
+ uk.ac.stfc.isis.ibex.targets;bundle-version="1.0.0"
```

Comment:
This is a minimum version but I think the difference is whether it is right click add and add in manifest.